### PR TITLE
create namespace scope controller

### PR DESCRIPTION
**Pull request by horis233**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

1. Please confirm that if this PR changes any image versions, then that's the sole change this PR makes.

**Release note**:

```release-note

```

### CODE REVIEW

**Comment by horis233** [comment]
@bitscuit @Daniel-Fan Could you help review this PR.
It is a cherry-pick from #609

File: controllers/common/fetch.go [hunk 1]
```diff
@@ -46,6 +46,35 @@ func FetchOperandRegistry(c client.Client, key types.NamespacedName) (*apiv1alph
 	return reg, nil
 }
 
+// FetchALLOperandRegistry lists the OperandRegistry instance with default value
+func FetchALLOperandRegistry(c client.Client, label map[string]string) (*apiv1alpha1.OperandRegistryList, error) {
```
Comment (bitscuit) [inline controllers/common/fetch.go:50]: why not name the function the same as master branch PR one https://github.com/IBM/operand-deployment-lifecycle-manager/pull/609/files#diff-7b924b0b44ff7b9702a981217edb4f0fd0bb4ec3205ca3f5ebd54b895f9a0d32R75-R76?

File: controllers/namespacescope_controller.go [hunk 2]
```diff
@@ -0,0 +1,240 @@
...
+
+	nssv1 "github.com/IBM/ibm-namespace-scope-operator/api/v1"
+
+	operatorv1alpha1 "github.com/IBM/operand-deployment-lifecycle-manager/api/v1alpha1"
+	fetch "github.com/IBM/operand-deployment-lifecycle-manager/controllers/common"
+	"github.com/IBM/operand-deployment-lifecycle-manager/controllers/constant"
+	"github.com/IBM/operand-deployment-lifecycle-manager/controllers/util"
+)
+
+// NamespaceScopeReconciler automagically updates NamespaceScope CR with the proporly namespace
```
Comment (bitscuit) [inline controllers/namespacescope_controller.go:45]:
```suggestion
// NamespaceScopeReconciler automagically updates NamespaceScope CR with the proper namespace
```
fix spelling

**Review by bitscuit — CHANGES_REQUESTED** [review]

File: controllers/common/fetch.go (hunk 1 above)
Comment (horis233) [reply controllers/common/fetch.go:50]: I want to keep the function name being consistent with other fetch functions in 1.4

**Review by bitscuit — APPROVED** [review]
/lgtm

**Pull request merged**